### ISSUE TITLE
App does not launch

### ISSUE THREAD

**Issue by hellomiakoda** [iabem97/komorebi#112]
Ubuntu 18.04 64-bit
Nvidia graphics on proprietary drivers

I click to launch the app, wait icon appears as the cursor, and then.... nothing

**Comment by pokemon4e** [comment]
try installing libgtop-2.0.so.10 from debians site https://packages.debian.org/stretch/amd64/libgtop-2.0-10/download

**Comment by TechieAndroid** [comment]
Here are my errors after fixing libgtop:
`** (komorebi:10056): CRITICAL **: 10:28:23.629: file /home/abe/Videos/komorebi/build/src/OnScreen/DesktopIcons.c: line 541: uncaught error: Key file does not have key “Exec” in group “Desktop Entry” (g-key-file-error-quark, 3)
Reading config file..

** (komorebi:10056): CRITICAL **: 10:28:23.851: string_contains: assertion 'self != NULL' failed
`

**Comment by thedonquixotic** [comment]
@TechieAndroid @pokemon4e @hellomiakoda This has to do with an issue related to the deprecation of realpath in Ubuntu 17 and up I believe. There are deb packages you can download and install outside of core to get Kororembi working again. @iabem97 seems to have moved on to other projects, so if it's okay with him, I plan on forking this and helping push further development and maintenance of the project. If you are interested in helping me let me know.

**Comment by thedonquixotic** [comment]
Also this issue should be closed.

**Comment by TechieAndroid** [comment]
@thedonquixotic I agree. Actually I got it working perfectly fine in 18.04 LTS after compiling it from scratch. It is a little quirky but it works fine. I'd be glad to help out on it, I know a little bit of programming in Python and C++ (I'm still learning) but I'm better at scripting. I know some things on audio and video codecs and could possibly help out there, I could use some experience to learn even more though.

**Comment by thedonquixotic** [comment]
@TechieAndroid Hey the git hub owner @iabem97  actually got our messages and he's completed the pull requests that had been waiting for approval. The newest version works great! I think there's some more features or improvements that could be made to it, but I think it's worth contributing if you can.

**Comment by zwhitchcox** [comment]
2.1 does not work on focal

**Comment by Ev1lbl0w** [comment]
This issue has been fixed on our [fork](https://github.com/Komorebi-Fork/komorebi). You'd have to compile it, but we're planning to release a v2.2.0 tomorrow, if you want to wait for it.

**Comment by zwhitchcox** [comment]
Oh ok, cool!

**Comment by Mujtahid357** [comment]
app is not work properly, even it is not open
